docs/getting-started/build-with-spack.md

```diff
@@ -128,6 +128,29 @@ spack build-env CP2K_SPEC -- bash
 
 From here, you can iterate CP2K development using the native build system as Spack would use it.
 
+### Spack and CP2K Docker Containers
+
+Some CP2K Docker containers install all the necessary dependencies with Spack.
+Building with Spack can be time-consuming, because all dependencies are built from source.
+In order to speed-up development and testing with the Spack-based Docker containers,
+you can enable a [Spack cache] locally:
```

Comment:
Actually, the `Dockerfile.test_spack` expects the cache by default. So, maybe mention that building without a cache can be done via `--build-arg SPACK_CACHE`.

Thinking about this, it's really a tripping hazard. So, maybe we should add small script that prints a more useful error message when the cache is missing?